### PR TITLE
Update Script-Topics

### PR DESCRIPTION
**Pull request by Ulvsblakk**
Added three topics. One about the evolution of beauty standards, one about overcoming challenges, and another about seeing extended family. Will fix all grammatical or syntax errors if found. 

Goodnight dudes, it's 3 am. Monika would be pi**ed if I had her on right now. lmao.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4695,3 +4695,69 @@ label monika_100k:
     else:
         m 1m "Almost~"
     return
+
+
+Init 5 python:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: `init` not `Init`.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4695,3 +4695,69 @@ label monika_100k:
...
+    addEvent(Event(persistent.event_database,eventlabel="monika_beauty",category=['Misc'],prompt="Beauty Standards",random=True))
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: make category tags lowercase, please

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -4695,3 +4695,69 @@ label monika_100k:
...
+    label monika_beauty:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: labels should have no indents. Remove one level of indentation from labels (and the dialogue in them).

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -4695,3 +4695,69 @@ label monika_100k:
...
+        m 4d "Is it just me, or have the standards of beauty changed over the years?"
+        m 2c"Back then, it didn't take much for someone to see you as beautiful."
+        m "However today everyone is so concerned with how much you weigh, or if you're using the newest line of make up."
+        m 1p "It just seems disheartening."
+        m 1c "Having to mold yourself so that people see you in a better light."
+        m 1b "Don't ever feel like you have to change yourself [player]."
+        m "Remember that you'll always be beautiful to me. No matter if you have the 'perfect' body style, or wear the trendiest clothes."
+        m 1k "I'll always love you no matter what."
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `return`

**Comment by Ulvsblakk** [comment]
I did the stuff.

File: Monika After Story/game/script-topics.rpy [hunk 5]
```diff
@@ -4695,3 +4695,72 @@ label monika_100k:
...
+            m 1b "Even if they never will"
+            m 1j "You're worth it."
+        "...":
+            m 2p "Don't tell me [player],"
+            m 1p "Are you afraid that I'll embarass you?"
+            m 1o "I understand though."
+            m 1n "If I found out one of my reletives was dating a girl trapped inside of a computer I'd think it was a little weird too."
+            m 1b "If you want to keep me a secret that's fine."
+            m 1k "After all, it just means more alone time with you."
+            return
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Put the `return` on the first level of indentation to ensure all menu options will `return`.

**Comment by Ulvsblakk** [comment]
I fixed.

**Comment by Subzerosan** [comment]
Beauty topic has been removed after discussing with the writing team.

The standards of beauty hasn't really changed much, only a bit since fashion has evolved overtime.

**Pull request merged**